src/Database/Connection.php

```diff
@@ -158,10 +159,11 @@ public function driver($driver = null, $config = [])
             return $this->_driver;
         }
         if (is_string($driver)) {
-            if (!class_exists($driver)) {
+            $className = App::className($driver, 'Database/Driver');
+            if (!$className || !class_exists($className)) {
```

Comment:
We could make `App::className()` call class_exists() in the fully qualified namespace case too. Then we wouldn't need any of the `class_exists()` calls related to classname resolution.